### DISCUSSION TITLE
Using 'Form' inside a modal?

### DISCUSSION THREAD

**Discussion by ghost** [adobe/react-spectrum#6481] [Q&A] [answered]
Hey there,
since inside dialogs (in my case, a modal) the `<Form>` element would need to be inside `<Content>`, I cannot access its data / `onSubmit` when using a `<ButtonGroup>` (as recommended in the docs) - what's the favorable approach for using forms in modals?

Thanks in advance, any pointers much appreciated 🙏

**Comment by snowystinger** [comment, marked as answer]
This would probably solve it. Would you like to contribute it https://github.com/adobe/react-spectrum/issues/1456 ?

**Reply by ghost** [reply]
Strange how I missed that .. I'll head over there and see what's what, thanks 👍
